changelog-generator/content/changelogs/20230809_files_service-1-1-0.md

```diff
@@ -0,0 +1,23 @@
+---
+service: files-service
+version: 1.1.0
+date: 2023-08-09
+---
+
+**🎁 Features**
+* Added File Service settings
+  * A new settings property `disableForceDownloadForMimeTypes` may be configured to override the File Service’s default behavior of enforcing file downloads and enable viewing of files directly in the browser for selected mime types.
+
+
+**⚒️ Improvements**
+* Error messages
+  * Improved the error message for creating a file without a request body
+
+
+**🐞 Bugs Fixed**
+
+* File Tags
```

Comment:
I think bug fixes don't have this title like line. Normally it is just a list without indentations?
Not sure if it matters a lot.
<img width="513" alt="Screenshot 2023-08-09 at 10 00 46" src="https://github.com/ExtraHorizon/docs/assets/93595259/54c3f6d8-45f2-4a63-9e66-b9f8e03c743c">

Reply:
I also think the improvements can be just one line :

* Improved the error message for creating a file without a request body

The * Error messages is unnecessary

Reply:
I just copied the release notes and changed the format slightly, I'll out it to this format ^^

Reply:
Yeah, I should have noticed during the review of the release notes on confluence. 😬 Here it stuck out a bit more.